docs-master/Schema.md

```diff
@@ -88,6 +88,30 @@ However, if you rarely query all comments by its author, indexing `author_id` is
 
 In general, most `_id` fields are indexed. Sometimes, `boolean` fields are worth indexing if you often use it for queries. However, you should almost never index date (`_at`) columns or `string` columns.
 
+## Advanced
+
+### Unsafe SQL schema
+
+If you want to modify the SQL used to set up the SQLite database, you can pass `unsafeSql` parameter
+to `tableSchema` and `appSchema`. This parameter is a function that receives SQL generated by Watermelon,
```

Comment:
I think this change indirectly makes SQL generated by Watermelon kinda a public API, since now apps can search and replace specific part etc. What will be our approach to changes in generated SQL? Will these be marked as breaking changes? If not maybe we should add this info to documentation to avoid surprises.

Reply:
I see your point, but I don't want to overcomplicate this. IMO `unsafe` means unsafe, can break at any moment. But I agree it's worth noting that in the docs, and we can make it a habit to note encode schema changes in changelog (but not as breaking changes in the semver sense)

Reply:
Yeap, seems like a reasonable approach :)